Makefile

```diff
@@ -1,4 +1,5 @@
 WORKSPACES_WITH_RUST_MODULES := $(wildcard $(addsuffix /crates, $(wildcard ./packages/*)))
+PEER_ID := $(shell eval ./scripts/get-hopr-address.sh "$(api_token)" "$(endpoint)")
```

Comment:
This command execution depends on variables which are target specific. Therefore, the best place to put this is in the target itself, since this is not a globally usable function.

Reply:
Removed and changed into target specific